ast/functions.go

```diff
@@ -687,6 +687,8 @@ const (
 	AggFuncStddevSamp = "stddev_samp"
 	// AggFuncJsonObjectAgg is the name of json_objectagg function
 	AggFuncJsonObjectAgg = "json_objectagg"
+	// AggFuncApproxUniq is the name of approx_uniq function.
+	AggFuncApproxUniq = "approx_uniq"
```

Comment:
what's the function name used in other databases?

Reply:
clickhouse use the name `uniq` for the same function and use `uniqExact` for count distinct.

Reply:
how about hive, spark, impala?

Reply:
@ilovesoup

Reply:
presto: approx_distinct
spark: approx_count_distinct
impala: no special function but has a switch named appx_count_distinct
I suggest we named like approx_distinct for fewer keystrokes.

Reply:
MS SQL Server: [APPROX_COUNT_DISTINCT](https://docs.microsoft.com/en-us/sql/t-sql/functions/approx-count-distinct-transact-sql)

Oracle: [APPROX_COUNT_DISTINCT](https://docs.oracle.com/en/database/oracle/oracle-database/20/sqlrf/APPROX_COUNT_DISTINCT.html#GUID-50055A05-0187-4481-AFE5-2414F7227713)

BigQuery: [APPROX_COUNT_DISTINCT](https://cloud.google.com/bigquery/docs/reference/standard-sql/approximate_aggregate_functions)

Reply:
make function name: `APPROX_COUNT_DISTINCT` just like oracle and microsoft.

Reply:
@zz-jason @ilovesoup @lzmhhh123 PTAL